### DISCUSSION TITLE
Option to report specific photos when swiping.

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#112] [Ideas & Features]
For example, when you are swiping through some random person's profile, and you can tell that it is photoshopped or has a heavy filter, then you can report the specific photo. And if enough people do that, then they need to replace it.

**Comment by Oshyan** [comment]
Definitely. I think many apps already do this, if not for obvious edits, than at least for inappropriate content, etc.

**Comment by WilliamDormer** [comment]
This has been added as an issue for the flutter application repo

**Comment by ajbott** [comment]
Being able to easily report regarding various topics is going to be a huge part of the app enforcement. This allows us to easily identify issues, categorize their severity, prioritize accordingly, and punish fairly.
